GeometryTools/Size.swift

```diff
@@ -25,9 +25,20 @@ public struct Size {
         self.height = height
     }
 
+    /// - Returns: a `Size` with both dimensions scaled by the same value.
     public func scaled(by value: Double) -> Size {
         return Size(width: width * value, height: height * value)
     }
+
+    /// - Returns: a `Size` scaled by the given dimensions.
+    public func scaled(byWidth widthScale: Double = 1, byHeight heightScale: Double = 1) -> Size {
+        return Size(width: width * widthScale, height: height * heightScale)
```

Comment:
Or, maybe its smarter to go the other way around. Ultimately, I assume the compiler eats all of this up and its all the same.